utils/src/test/kotlin/UtilsTest.kt

```diff
@@ -182,6 +182,12 @@ class UtilsTest : WordSpec({
 
             filterVersionNames("1.11.6", names).joinToString("\n") shouldBe "1.11.6"
         }
+
+        "find names with only a single revision number as the version" {
```

Comment:
I'm not 100% sure which of the following would now be successfully handled,
would it be worth adding some of the below as test case?

my_project_v1-2.0.1-2
my_project_v1-2.1-2
my_project_v1-2-2

Reply:
Sure, feel free to do so 😁